docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/authentication-and-authorization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Authentication and authorization
+description: Information about Glue API authentication and authorization.
+last_updated: Oct 23, 2022
+template: glue-api-storefront-guide-template
+---
+
+For the authentication, Spryker implements the [OAuth 2.0 mechanism](https://tools.ietf.org/html/rfc6749). On the REST API level, it is represented by the Login API.
+
+To get access to a protected resource, a user obtains an *access token*. An access token is a JSON Web Token used to identify a user during API calls. Then, they pass the token in the request header.
+
+![auth-scheme.png](https://spryker.s3.eu-central-1.amazonaws.com/docs/Glue+API/Glue+API+Storefront+Guides/Authentication+and+Authorization/auth-scheme+%281%29.png)
+
+For security purposes, access tokens have a limited lifetime. When retrieving an access token, the response body also contains the token's lifetime, in seconds. When the lifetime expires, the token can no longer be used for authentication.
+
+There is also a *refresh token* in the response. When your access token expires, you can exchange the refresh token for a new access token.  The new access token also has a limited lifetime and a new refresh token.
+
+The default lifetime of the access tokens is 8 hours (28800 seconds) and 1 month (2628000 seconds) of the refresh tokens.
+
+For security purposes, when you finish sending requests as a user, or if a token gets compromised, we recommend revoking the refresh token. Revoked tokens are marked as expired on the date and time of the request and can no longer be exchanged for access tokens.
+
+Expired tokens are stored in the database, and you can configure them to be deleted.
+
+## Authentication in the Storefront and Backend applications
+
+Below, you can find information about [Backend and Storefront API module differences](/docs/scos/dev/glue-api-guides/{{page.version}}/glue-backend-api/how-to-guides/backend-and-storefront-api-module-differences.html) and [Authentication integration.](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-backend-api/glue-api-authentication-integration.html)
```

Comment:
Do we have this doc already (backend-and-storefront-api-module-differences.html)?

Reply:
yes

Reply:
Cool